static/javascript/launch.js

```diff
@@ -543,6 +542,16 @@ function loadSupplementaryDataInfo() {
   }
 }
 
+function clearSDSMetadata() {
+  let dataset = document.querySelectorAll("#sds_data > div");
```

Comment:
I think it would be better to use something a bit more descriptive here than "div". Same with "div_array"